app/assets/stylesheets/locals/call_to_action.css.scss

```diff
@@ -25,6 +25,8 @@
     }
     .well {
         width: 100%;
+        min-height: 120px;
+        
```

Comment:
@mccalluc re: units.. i've been convinced to ditch `.em` altogether. I can't think of a good use case now that modern browsers handle resizing as well as they do. And for that matter, using anything except `px` seems to need a special reason.